src-web/scss/common.scss

```diff
@@ -88,6 +88,10 @@ div.disable-tooltip {
   cursor: not-allowed;
 }
 
+span.table-status-label {
```

Comment:
should this be in `policy-details-overview.scss` instead ? is this supposed to be global?

Reply:
I made it global since I figured we'd always want padding between the icon and the text, but I can definitely keep it inside `policy-details-overview.scss` if that's preferable.

Reply:
ok. that makes sense